project/Build.scala

```diff
@@ -249,6 +252,10 @@ object Build {
           "-encoding", "utf8"
       ),
 
+      scalastyleCheck := Def.taskDyn {
```

Comment:
Does this really need to be `dyn`? Ugh. (I have never used `toTask` on an `InputTask`, so I'm genuinely asking).